server/src/main/java/org/eclipse/lsp/cobol/core/engine/CobolLanguageEngine.java

```diff
@@ -132,7 +137,19 @@ public CobolLanguageEngine(
       Node rootNode = syntaxTree.get(0);
       SyntaxTreeEngine syntaxTreeEngine = new SyntaxTreeEngine(rootNode);
       accumulatedErrors.addAll(syntaxTreeEngine.processTree());
-      context = context.toBuilder().rootNode(rootNode).build();
+      // This is a temporal solution only for compatibility
+      // Definitions, usages and variabled are set here for "Go to definition" feature and others
+      List<Variable> definedVariable = rootNode.getDepthFirstStream()
```

Comment:
definedVariables  or definedVariableList

Reply:
Indeed!

Reply:
> Indeed!

Just to be consistent in all place, e.g. in` ProgramNode` definedVariables